client/protocols/analytics.py

```diff
@@ -21,6 +25,70 @@ def run(self, messages):
             # TODO(denero) Get the canonical name of the question
             statistics['question'] = self.args.question
 
+        statistics['started'] = self.check_start(messages['file_contents'])
+        print("DEBUG:" + statistics['started'])
+
         messages['analytics'] = statistics
 
+    def check_start(self, files):
+        """returns a dictionary where the key is question number, and the value
```

Comment:
This docstring leads me to believe the key is an `int` or a `float`, but shouldn't it be a string of whatever the Question name is?